<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurnOrderWidget</class>
 <widget class="QWidget" name="TurnOrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turn Order List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../logo-lr.qrc">
    <normaloff>:/Logo/rpgassistant.ico</normaloff>:/Logo/rpgassistant.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableTurnOrder">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>400</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>INI</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>INI mod.</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>cHP</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HP</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mod.</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutActions">
     <item>
      <widget class="QLabel" name="labelCurrentIni">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStart">
       <property name="text">
        <string>New &amp;round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDamage"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDamage">
       <property name="text">
        <string>Da&amp;mage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text">
        <string>Ne&amp;xt</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutIniMod">
     <item>
      <widget class="QLabel" name="labelIniMod">
       <property name="text">
        <string>INI mod.</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxIniMod</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonIniModHide">
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxIniMod">
       <property name="minimum">
        <number>-99</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonIniModApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonWait">
       <property name="text">
        <string>&amp;Wait</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAdd">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>&amp;Name</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item>
      <widget class="QLabel" name="labelIni">
       <property name="text">
        <string>&amp;INI</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxIni</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxIni">
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLe">
       <property name="text">
        <string>&amp;HP</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxLe</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxLe">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutListActions">
     <item>
      <widget class="QPushButton" name="buttonLoad">
       <property name="text">
        <string>&amp;Load list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>&amp;Save list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>&amp;Clear list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../logo-lr.qrc"/>
 </resources>
 <connections/>
</ui>
